app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/details/DetailViewExampleActivity.java

```diff
@@ -24,9 +24,27 @@
  */
 public class DetailViewExampleActivity extends Activity {
 
+    /***
+     * True if using trailer as a background.
+     */
+    protected boolean hasBackgroundVideo() {
```

Comment:
Created a new DetailsFragment called "DetailViewExampleWithVideoBackgroundFragment" for DetailView example with video background.